src/data/drinkRecipes/autumn.ts

```diff
@@ -0,0 +1,78 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/gEsRZdHlTm7j2rVHC-_QkyxZUSw=/720x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/autumn-720x720-primary-357cede4ffa9487f93136f76b21d8591.jpg"
+
+const autumn: DrinkRecipe = {
+  name: "Autumn",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Applejack",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Amontillado Sherry",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pear Brandy",
+      unit: "oz",
+      unitAmount: 0.25,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Maple Syrup Grade A",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Apple Bitters",
+      unit: "Dash",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [],
+  portionlessIngredients: [],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Allspice",
+      suffix: "Tincture",
```

Comment:
Should Tincture be part of the ingredient here? I think Allspice and Allspice tincture are totally different (based on googling - you tell me).

Reply:
If you have "Allspice", what do you need to do to it to make it into "Allspice Tincture"? It looks like Allspice is a powder and allspice tincture is a liquid.

Reply:
IMHO I think the ingredient should be "Allspice Tincture" because the prep of soaking it for a set period of time isn't something you can just knock out (it requires a set period of time). I'd prefer to just buy the tincture not "grow" my own.

I'll leave it up to you.